docs/labs.md

```diff
@@ -109,5 +109,5 @@ In the event the new room list needs to be disabled, add this to your config.jso
 }
 ```
 
-**Note**: In a future release the new room list will not be optional.
```

Comment:
I think you need to revert these lines, from the conflict, since you're not trying to change them.